### PR TITLE
FIX CSS and JS warnings

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Description
- Suppressed  `eslint` and `sass-lint` warnings
- Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.
   -  Replaced divider `/` to `calc(*/*)` function

### Parent issue
- https://github.com/silverstripe/silverstripe-admin/issues/1416

### CODE REVIEW

File: client/src/styles/WorkflowField.scss [hunk 1]
```diff
@@ -57,8 +57,8 @@
           background: -o-linear-gradient($white, $grey-lightish);
           background: -ms-linear-gradient($white, $grey-lightish);
           background: linear-gradient($white, $grey-lightish);
-          -webkit-text-shadow: $white 0 1px 1px;
-          -moz-text-shadow: $white 0 1px 1px;
+          -webkit-text-shadow: $white 0 1px 1px; // sass-lint:disable-line no-misspelled-properties
+          -moz-text-shadow: $white 0 1px 1px; // sass-lint:disable-line no-misspelled-properties
```
Comment (maxime-rainville) [inline client/src/styles/WorkflowField.scss]: This is properly supported by all browsers. I don't think we should keep the old CSS properties.

```suggestion
```

File: client/src/styles/AdvancedWorkflowAdmin.scss [hunk 2]
```diff
@@ -15,11 +15,11 @@
 .cms-tree.jstree {
   span.badge.status-embargo,
   span.badge.status-expiry,
-  span.badge.status-embargo_expiry {
+  span.badge.status-embargo_expiry {// sass-lint:disable-line class-name-format
```
Comment (maxime-rainville) [inline client/src/styles/AdvancedWorkflowAdmin.scss:18]: I would be fined if we updated those classes in the actual HTML so we don't have to use bad call names and ids in the CSS.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Made some minor suggestion, but I'm not overly fuss about those.

File: client/src/styles/AdvancedWorkflowAdmin.scss (hunk 2 above)
Comment (sabina-talipova) [reply client/src/styles/AdvancedWorkflowAdmin.scss:18]: I leave it as is?!

File: client/src/styles/WorkflowField.scss (hunk 1 above)
Comment (sabina-talipova) [reply client/src/styles/WorkflowField.scss]: DONE

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**